projects/dev-schematics/src/shared/utils/package-utils.ts

```diff
@@ -1,5 +1,5 @@
 import { Tree } from '@angular-devkit/schematics';
-import { SPARTACUS_CORE, UTF_8 } from '../constants';
+import { SPARTACUS_CORE, UTF_8 } from '@spartacus/schematics/public_api';
```

Comment:
We shouldn't specify `public_api`, it should be only `@spartacus/schematics`. Does it work without the `public_api` part?

Reply:
Solved with approach yb adding public_api.js file to "main" field in package.json